rero_mef/monitoring/tasks.py

```
@@ -0,0 +1,134 @@
+                "query": {"term": {"node.id": node_id}},
+                "_source": [
+                    "@timestamp",
+                    "node.indices.search_query_total",
+                    "node.indices.index_total",
+                    "node.indices.index_failed_total",
+                ],
+            },
+        )
+        hits = result["hits"]["hits"]
+        return hits[0]["_source"] if hits else None
+    except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)